### ISSUE TITLE
The inference can only output 300 boxes

### ISSUE THREAD

**Issue by n9705** [ultralytics/yolov5#1082]
## ❔Question
Hi, I apply yolov5 to detect single Chinese chars. All my training data is ok, but when doing inference, I found the network can only output most 300 boxes an image. Does someplace limit the max-output number?

## Additional context
`Fusing layers... 
Model Summary: 407 layers, 8.84337e+07 parameters, 0 gradients
image 1/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/1.png: 128x640 300 chars, Done. (0.032s)
image 2/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/16.png: 640x640 300 chars, Done. (0.049s)
image 3/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/2.png: 128x640 300 chars, Done. (0.028s)
image 4/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/3.png: 192x640 300 chars, Done. (0.029s)
image 5/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/6.png: 640x640 88 chars, Done. (0.045s)
image 6/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/7.png: 128x640 300 chars, Done. (0.027s)
image 7/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/8.png: 576x640 300 chars, Done. (0.045s)
image 8/8 /home/fzuirdata/wjy/newspaperOCR/yolov5/data/test_data/zztj202009171352001_页面_0001.tif: 640x448 13 chars, Done. (0.036s)
Results saved to inference/output
Done. (1.002s)
`

**Comment by n9705** [comment]
Oh, I thought I found the place limits the max detect-num. It is at utils.general.py, line 603:
`max_det = 300  # maximum number of detections per image`

**Comment by glenn-jocher** [comment]
@n9705 yes, you simply need to modify this parameter, which is set at 300 by default.
